core-services/prow/02_config/_plugins.yaml

```diff
@@ -2074,6 +2074,8 @@ plugins:
   - welcome
   - wip
   - yuks
+  openshift-cnv/cnv-ci:
```

Comment:
it is better to have a config for the whole organization

Reply:
I have updated the file. The webhooks and the robots are configured at the moment on repo level. cnv-ci is our only repo at this moment.